move-to-sdk/app/io/sphere/sdk/facets/RangeFacet.java

```diff
@@ -0,0 +1,59 @@
+package io.sphere.sdk.facets;
+
+import io.sphere.sdk.search.PagedSearchResult;
+import io.sphere.sdk.search.RangeFacetResult;
+import io.sphere.sdk.search.model.RangeTermFacetedSearchSearchModel;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+/**
+ * Facet that contains a list of range options.
+ * @param <T> type of the resource for this facet (e.g. ProductProjection)
+ */
+public interface RangeFacet<T> extends Facet<T> {
+
+    /**
+     * Gets the complete options without limitations.
+     * @return the complete possible options
+     */
+    List<FacetOption> getAllOptions();
+
+    /**
+     * Obtains the truncated options list according to the defined limit.
+     * @return the truncated options if the limit is defined, the whole list otherwise
+     */
+    List<FacetOption> getLimitedOptions();
```

Comment:
Another things, `limitedOptions` and `allOptions` are necessary concepts for select facets, where you need to somehow control the potentially too large amount of values coming from the platform. But in bucket range facets you know very well the amount, so this mechanism is not useful here.